scripts/run-clang-format

```diff
@@ -7,6 +7,13 @@ if [ -z "$CLANG_FORMAT_BINARY" ] ; then
   echo "Failed to find clang-format binary in path: $PATH"
   exit 1
 fi
+CLANG_FORMAT_VERSION=$($CLANG_FORMAT_BINARY --version)
+if [[ "$CLANG_FORMAT_VERSION" == *"5.0"* ]]; then
```

Comment:
How does this sync with what @pmiettinen was seeing?

Reply:
It doesn't which is honestly the strangest part for me but travis reports 5.0.0 which looks compatible with what I was using locally (5.0.1)

Reply:
Seems fine on my setup now (no unexpected modifications):

```
pasi@pasi-linux:~/swiftnav/piksi_buildroot$ git checkout benaltieri/add_format_documentation
M       buildroot
Branch benaltieri/add_format_documentation set up to track remote branch benaltieri/add_format_documentation from origin.
Switched to a new branch 'benaltieri/add_format_documentation'
pasi@pasi-linux:~/swiftnav/piksi_buildroot$ make clang-format
Incorrect formatter version (clang-format version 6.0.0-1ubuntu2~16.04.1 (tags/RELEASE_600/final)), expected 5.0
Makefile:272: recipe for target 'clang-format' failed
make: *** [clang-format] Error 1
pasi@pasi-linux:~/swiftnav/piksi_buildroot$ sudo update-alternatives --config clang-format
[sudo] password for pasi:
Sorry, try again.
[sudo] password for pasi:
There are 3 choices for the alternative clang-format (providing /usr/bin/clang-format).

  Selection    Path                                Priority   Status
------------------------------------------------------------
  0            /usr/lib/llvm-6.0/bin/clang-format   100       auto mode
  1            /usr/lib/llvm-4.0/bin/clang-format   100       manual mode
  2            /usr/lib/llvm-5.0/bin/clang-format   100       manual mode
* 3            /usr/lib/llvm-6.0/bin/clang-format   100       manual mode

Press <enter> to keep the current choice[*], or type selection number: 2
update-alternatives: using /usr/lib/llvm-5.0/bin/clang-format to provide /usr/bin/clang-format (clang-format) in manual mode
pasi@pasi-linux:~/swiftnav/piksi_buildroot$ make clang-format
Formatting C/C++ Sources with clang-format version 5.0.0-3~16.04.1 (tags/RELEASE_500/final)
pasi@pasi-linux:~/swiftnav/piksi_buildroot$ git status
On branch benaltieri/add_format_documentation
Your branch is up-to-date with 'origin/benaltieri/add_format_documentation'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)

        modified:   buildroot (untracked content)

no changes added to commit (use "git add" and/or "git commit -a")
pasi@pasi-linux:~/swiftnav/piksi_buildroot$
```